common/components/SettingsForm.tsx

```diff
@@ -26,6 +26,7 @@ import { AsbplayerSettings, KeyBindName, SubtitleListPreference } from '@project
 import { computeStyles, download } from '@project/common/util';
 import { CustomStyle, supportedLanguages, validateSettings } from '@project/common/settings';
 import { useOutsideClickListener } from '@project/common/hooks';
+import { styled } from '@material-ui/core';
```

Comment:
Can you import it like this instead?
```import styled from '@material-ui/core/styles/styled';```
This will reduce the amount of code imported as the entire `@material-ui/core` module won't be imported.